### DISCUSSION TITLE
Vulnerability issue in dropwizard.assets 2.1.1

### DISCUSSION THREAD

**Discussion by zZToasterZz** [dropwizard/dropwizard#5867] [General]
After running the OWASP Dependency checker against the dropwizard, one vulnerability is raised against dropwizard-assets dependency which is brought in:
jakarta.servlet-api-4.0.4.jar CVE-2022-31569
<img width="485" alt="image" src="https://user-images.githubusercontent.com/79910936/190059558-91e46c10-b961-47d8-9dc1-e8a02bc23eb4.png">

**Comment by joschi** [comment]
Dropwizard 2.1.2 has been released recently: https://github.com/dropwizard/dropwizard/releases/tag/v2.1.2

Also, [CVE-2022-31569](https://nvd.nist.gov/vuln/detail/CVE-2022-31569) has been rejected:

> CVE has been marked "REJECT" in the CVE List. These CVEs are stored in the NVD, but do not show up in search results.
